clang/lib/Sema/SemaTemplateInstantiate.cpp

```diff
@@ -346,6 +351,11 @@ MultiLevelTemplateArgumentList Sema::getTemplateInstantiationArgs(
     CurDecl = Response::UseNextDecl(ND).NextDecl;
   }
 
+  if (!ND) {
+    assert(DC);
```

Comment:
Does this assertion makes sense? If `ND` and `DC` both were null, the first assertion in this function would have failed already.

Reply:
Probably not necessary, I'll remove it.  I think I added that top assert later on.